modules/metrics/metrics.go

```diff
@@ -71,14 +75,23 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 
 // UnmarshalCaddyfile sets up the handler from Caddyfile tokens. Syntax:
 //
-//     metrics <matcher>
+//     metrics <matcher> {
```

Comment:
Technically the matcher is optional, so it should be:

```suggestion
//     metrics [<matcher>] {
```

Reply:
Ah right. I had a bit of a head-scracth moment because the entire block is optional too, but there seems not to be a convention for specifying that?

Reply:
Fixed